src/apis/vaults.ts

```diff
@@ -18,15 +20,30 @@ export interface VaultsAPI {
 }
 
 export class DefaultVaultsAPI {
+    private issueAPI!: DefaultIssueAPI;
     granularity = 5;
 
-    constructor(private api: ApiPromise) { }
+    constructor(private api: ApiPromise) { 
+    }
 
     async list(): Promise<Vault[]> {
         const vaultsMap = await this.api.query.vaultRegistry.vaults.entriesPaged({ pageSize: 1 });
         return vaultsMap.map((v) => v[1]);
     }
 
+    async listIssueRequests(vaultId: AccountId): Promise<IssueRequest[]> {
```

Comment:
Also, rename this function to `mapForVault`

Reply:
The name might be misleading unless the programmer looks at the entire function signature - it's unclear that the map contains issue requests (it might as well contain redeem requests). Maybe a more suggestive name would be `mapIssueRequests`

Reply:
We can leave it like this for now and I'll rename in the PR for vault redeem requests